.github/ISSUE_TEMPLATE.md

```diff
@@ -1,9 +1,14 @@
 <!--
 Thank you for reporting an issue.
 
-This issue tracker is for bugs and issues only.
-If you need general support please use 
-our mailing list. https://github.com/zfsonlinux/zfs/wiki/Mailing-Lists
+*IMPORTANT* - Please search our issue tracker *before* making a new issue.
+If you cannot find a similar issue, then create a new issue.
+https://github.com/zfsonlinux/zfs/issues 
+
+*IMPORTANT* - This issue tracker is for *bugs* and *issues* only.
+Please search the wiki and mailing lists archives 
+and only if you don't find an answer ask on mailing list.
```

Comment:
`Please search the wiki and the mailing list archives before asking questions on the mailing list.`?